WooCommerce/Classes/ViewRelated/Orders/Order Creation/CollapsibleProductCard.swift

```diff
@@ -138,31 +134,40 @@ private struct CollapsibleProductRowCard: View {
         CollapsibleView(isCollapsible: true,
                         isCollapsed: $isCollapsed,
                         safeAreaInsets: EdgeInsets(),
-                        shouldShowDividers: shouldShowDividers,
+                        shouldShowDividers: false,
+                        hasSubtleChevron: viewModel.hasParentProduct,
                         label: {
             VStack {
                 HStack(alignment: .center, spacing: Layout.padding) {
                     ProductImageThumbnail(productImageURL: viewModel.imageURL,
-                                          productImageSize: Layout.productImageSize,
+                                          productImageSize: viewModel.hasParentProduct ? Layout.childProductImageSize : Layout.parentProductImageSize,
                                           scale: scale,
                                           productImageCornerRadius: Layout.productImageCornerRadius,
                                           foregroundColor: Color(UIColor.listSmallIcon))
+                    .padding(.leading, viewModel.hasParentProduct ? Layout.childLeadingPadding : 0)
                     VStack(alignment: .leading) {
                         Text(viewModel.name)
+                            .font(viewModel.hasParentProduct ? .subheadline : .none)
+                            .foregroundColor(.black)
                         Text(viewModel.stockQuantityLabel)
-                            .foregroundColor(.secondary)
-                            .renderedIf(isCollapsed)
+                            .font(.subheadline)
+                            .foregroundColor(isCollapsed ? Color(.textSubtle) : .black)
                         Text(viewModel.skuLabel)
-                            .foregroundColor(.secondary)
+                            .font(.subheadline)
+                            .foregroundColor(.black)
```

Comment:
similar to above, `text` might work better in dark mode
```suggestion
                            .foregroundColor(.text)
```